crates/subspace-farmer/examples/plot-write-benchmark.rs

```diff
@@ -0,0 +1,60 @@
+use std::{path::PathBuf, sync::Arc};
+
+use clap::{Parser, ValueHint};
+use rand::prelude::*;
+use subspace_core_primitives::PIECE_SIZE;
+use subspace_farmer::Plot;
+
+#[derive(Debug, Parser)]
+struct Args {
+    /// In megabytes
+    #[clap(short, long, default_value = "8")]
+    buffer_size: u64,
+    /// In megabytes
+    #[clap(short, long, default_value = "1024")]
+    write_size: u64,
```

Comment:
Can we have number of pieces and batch size here instead, so it is easier to understand what values to put there?